### PR TITLE
games-*/*: updated eclasses, LICENSES, minor improvements

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This updates a few games related packages removing or updating mostly unused eclasses (eutils -> wrapper for example). Also updated a few LICENSES and made some minor improvements. All checked with `pkgcheck scan` too.

Please review
@thesamesam guess you're interested in this PR too :)

### CODE REVIEW

File: games-action/bzflag/bzflag-2.4.20.ebuild [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit autotools desktop flag-o-matic
+
+inherit autotools desktop
 
 DESCRIPTION="3D tank combat simulator game"
```
Comment (SoapGentoo) [inline games-action/bzflag/bzflag-2.4.20.ebuild:8]: l.71 change to
`find "${ED}" -name '*.la' -delete || die`

File: games-action/bzflag/bzflag-2.4.22.ebuild [hunk 2]
```diff
@@ -2,7 +2,8 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit autotools desktop flag-o-matic
+
+inherit autotools desktop
 
 DESCRIPTION="3D tank combat simulator game"
```
Comment (SoapGentoo) [inline games-action/bzflag/bzflag-2.4.22.ebuild:8]: ditto l.71 here too

**Review by SoapGentoo — CHANGES_REQUESTED** [review]
LGTM, let's just fix that ugly `.la` cleaner line

**Comment by mm1ke** [comment]
> LGTM, let's just fix that ugly `.la` cleaner line

Nice catch, just fixed the ebuilds.

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**